examples/bri-3/src/bri/identity/bpiSubjects/agents/bpiSubjects.agent.ts

```diff
@@ -74,4 +74,18 @@ export class BpiSubjectAgent {
 
     return bpiSubjectToDelete;
   }
+
+  public async fetchBpiSubjectsByIdAndThrowIfNoneExist(
+    bpiSubjectIds: string[],
+  ): Promise<BpiSubject[]> {
+    const bpiSubjects = await this.storageAgent.getBpiSubjectsById(
+      bpiSubjectIds,
+    );
+
+    if (bpiSubjectIds.length !== bpiSubjects.length) {
```

Comment:
not sure if we shall throw an exception, or log the ids which have data. 
@Therecanbeonlyone1969 what would you suggest?

Reply:
@Manik-Jain I will check out the behavior of the findMany used on the ORM side. Assuming that it will return a result as long as at least one record is found. If that is the case I think those found can be sent back to here in the code and I filter out the ones found and return them while logging the ones not found with an appropriate message. @skosito @ognjenkurtic @Ybittan what do you think?

Reply:
On the assumption that it will return results if there is a record found, I would agree filtering them out to return here and logging ones not found makes sense to me as a solution to ensure the right data goes through, while still properly logging what is missing.

Reply:
right

Reply:
Removed this functionality as its not being used anymore. will be added back in when invitation is done. @Manik-Jain @Ybittan